utils/plottools.py

```diff
@@ -11,11 +11,24 @@ def __init__(self, exp, figpath, dims, units, ticks=None):
         self.DIMS      = dims
         self.DIM_UNITS = units
         self.DIM_TICKS = ticks or self._default_dim_ticks()
+        if 't' in self.DIMS.keys():
+            self.update_time_mapper()
+
+    def update_time_mapper(self):
```

Comment:
This function can use `_update_time_mapper` ( add `_` in the head of the name) to suggest which is private in this class.

Reply:
When main program sets the ticks as minute, update_time_mapper() should be called to update the mapper dict so I keep it as a public function.
example codes are in the lines before the last plot of miles/test_draw_*.py